### DISCUSSION TITLE
vector_types.h: No such file or directory

### DISCUSSION THREAD

**Discussion by leizhenyuan** [NVIDIA/cutlass#1762] [Q&A]
Hi, i am getting start with cute, tried to include cute/Tensor.hpp, but met such build error below, can anyone give some comments?
Thanks in advance.

In file included from examples/cutlass/include/cute/config.hpp:155,
from examples/cutlass/include/cute/tensor_impl.hpp:44,
from examples/cutlass/include/cute/tensor.hpp:33,
from examples/layout.cpp:2:
examples/cutlass/include/cute/numeric/numeric_types.hpp:33:10: fatal error: vector_types.h: No such file or directory
   33 | #include <vector_types.h>
      |          ^~~~~~~~~~~~~~~~
compilation terminated.

Actually, i found this file in /usr/local/cuda/targets/x86_64-linux/include, but somehow looks like compiler didn't found it.

**Comment by thakkarV** [comment]
Are you using our cmake build system? If not, please do so and check if this persists

**Reply by leizhenyuan** [reply]
Thanks for your comment, i am using cmake system within cutlass, and i add a CMakeLists.txt within my own folder, it is quiet simple, just create a executable and include cute related files.

set(SOURCES_FILES layout.cpp)
add_executable(layout ${SOURCES_FILES})
file(GLOB_RECURSE CUTLASS_CUTE RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}/include include/cute/*.h*)
target_include_directories(layout PUBLIC  CUTLASS_CUTE)
